scripts/upgrade.sh

```diff
@@ -0,0 +1,203 @@
+#!/bin/bash
+
+#Author: Scality MetalK8s
+#
+#Summary: This script is used to upgrade Metalk8s cluster and all the components
+#          that make up the stack.
+#
+set -e
+set -u
```

Comment:
I think we want to keep the `set -o pipefail` (you removed what was around and that's good, you just removed a bit too much :slightly_smiling_face: )